### PR TITLE
REDCap env var re-org

### PR DESCRIPTION
**Pull request by tsibley**
The overarching motivation is an easier-to-manage system for associating credentials with projects that works consistently across our various codebases. The foundation for this was laid in previous core ID3C work, but most jobs/services kept using their existing env var organization until now. Details of the switch in the commits.

There are corresponding `redcap-env-reorg` branches in:

- [x] ID3C, https://github.com/seattleflu/id3c/pull/198 and https://github.com/seattleflu/id3c/pull/201
- [x] ID3C customizations, https://github.com/seattleflu/id3c-customizations/pull/181 (not strictly required)
- [x] Switchboard, https://github.com/seattleflu/scan-switchboard/pull/32
- [x] Documentation, https://github.com/seattleflu/documentation/pull/35

It's times like these I wish we had a majestic monolith!

Still to do before a merge/deploy:

- [x] Update deployed version of ID3C with `pipenv update`, once PRs are merged
- [ ] Update production envdirs. Locally I have a `id3c-production/env.d/redcap/` with secrets in place ready to move into production. I've audited and cross-checked all the usages several ways and believe I have everything covered.

### CODE REVIEW

**Review by kairstenfay — APPROVED** [review]

**Review by kschwabefry — APPROVED** [review]
Thanks for doing this work Thomas!

File: bin/uw-ehs-report/import-to-uw-ehs-transfer-redcap [hunk 1]
```diff
@@ -78,4 +51,4 @@ if __name__ == '__main__':
         num_batches = math.ceil(ehs_dataset_num_rows / redcap_post_batch_size)
 
         for batch in np.array_split(ehs_dataset, num_batches):
-            post_to_redcap(batch.to_json(orient='records'))
+            project.update_records(batch.to_dict(orient='records'), date_format = "MDY")
```
Comment (kairstenfay) [inline bin/uw-ehs-report/import-to-uw-ehs-transfer-redcap]: I suspect with a real payload here (i.e. a large one), you'll run into this error:
```
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.6/logging/handlers.py", line 936, in emit
    self.socket.send(msg)
OSError: [Errno 90] Message too long
```

Comment (kairstenfay) [reply bin/uw-ehs-report/import-to-uw-ehs-transfer-redcap]: I'm working on refactoring the `redcap-data-dictionary` POST request to use id3c's REDCap DAL, and I'm running into that error when `Project._fetch()` is logging a payload of 336KB.

Comment (tsibley) [reply bin/uw-ehs-report/import-to-uw-ehs-transfer-redcap]: Nod. Good point; we've seen that before too with other log lines. We should look into fixing this specific log message, but the logging error shouldn't impact the actual functionality of the code.

**Comment by tsibley** [comment]
Rebased onto latest master, to pick up Pipfile.lock prod changes.

**Comment by tsibley** [comment]
(and again, for the same reason)

File: bin/uw-ehs-report/import-to-uw-ehs-transfer-redcap (hunk 1 above)
Comment (tsibley) [reply bin/uw-ehs-report/import-to-uw-ehs-transfer-redcap]: https://github.com/seattleflu/id3c/pull/204

**Pull request merged**